scripts/config.pl

```diff
@@ -1,307 +1,7 @@
 #!/usr/bin/env perl
```

Comment:
I assume this needs to be
```suggestion
#!/usr/bin/env python
```